lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1690,6 +1698,7 @@ def __init__(self, container, parameters):
         self.parameters_map['device_write_bps'] = 'device_write_bps'
         self.parameters_map['device_read_iops'] = 'device_read_iops'
         self.parameters_map['device_write_iops'] = 'device_write_iops'
+        self.parameters_map['pids_limit'] = 'pids_limit'
```

Comment:
Sorry for not noticing this earlier, but this isn't necessary (and the four lines above aren't, either). The map is needed if the names are different (`expected_*` vs. `*`), and is not defined for most options. Can you remove this (and the four lines above)?

Reply:
Make sense. Thank you for letting me know. I have made the changes.

Reply:
Thanks!